README.md

```diff
@@ -0,0 +1,126 @@
+# SolidusAssetsProxy
+
+This [Solidus](https://github.com/solidusjs/solidus) library provides an easy solution for proxying a site's assets with a CDN or a an image processing service like [imgix](https://www.imgix.com/) and [ReSRC](https://www.resrc.it/).
```

Comment:
ReSRC was actually acquired by Fastly and the service is gonna be merged into it somehow. Just imgix is fine as an example.